include/solvers/flow_control.h

```diff
@@ -57,104 +153,29 @@ class FlowControl
   double         beta_n1     = 0;
   double         flow_rate_n = 0;
   double         flow_rate_1n;
-  double         area = 0;
+  double         area;
+
+  // Variables used to improve convergence
+  bool   adjusted         = false;
+  double threshold_factor = 1.01; // 1%
```

Comment:
Same as above. What does adjusted do? Please add detail on what variables do. Is this variable controlled from the parameter file? Is it really useful?

Reply:
Ok! "Adjusted" allows to enable to set the force value to 0 to get closer to the desired flow rate value. 
It's important if the head loss is to big otherwise every time there's no force, the value get too low.
It's useful also to be set at true when the flow rate go down with no force and then get below the value. It means that the flow rate reached the value and now need to be controlled by a calculated beta to keep the desired flow rate.
I said "If flow is reached with no force, adjusted is set to true" at line 77 of the .cc file, but I will give more information.